ultravox/data/datasets.py

```diff
@@ -961,6 +1002,7 @@ def __init__(self, args: VoiceDatasetArgs) -> None:
         dataset = self._load_audio_dataset(
             "fixie-ai/soda-audio", split=args.split.value
         )
+        dataset = SizedIterableDataset(dataset, 0)
```

Comment:
Setting the size here to 0 for now. The length doesn't matter for non GenericVoiceDataset types because they can't use epoch. Also, planning on getting rid of these classes/extending to use GenericVoiceDataset soon anyways.

Reply:
It does matter. Combined with the `Range` changes this code is gonna remove any non-generic datasets.

Reply:
Updated, the Range code.